Scala/Scala.sublime-syntax

```diff
@@ -194,31 +240,40 @@ contexts:
       push: class-tparams-brackets
     - match: '(?=([\{\n]|extends))'
       pop: true
+
   class-tparams-brackets:
     - match: '\['
       push: class-tparams-brackets
     - match: '\]'
       pop: true
-    - include: main
+    - match: this
```

Comment:
It should.  The normal use-case for it doesn't matter, but it could be a subset of another word.